edx_solutions_api_integration/models.py

```diff
@@ -156,3 +162,187 @@ class APIUser(User):
     class Meta(object):
         """ Meta attribute to make this a proxy model"""
         proxy = True
+
+
+class PasswordHistory(models.Model):
+    """
+    This model will keep track of past passwords that a user has used
+    as well as providing contraints (e.g. can't reuse passwords)
+    """
+    class Meta(object):
+        app_label = "student"
+
+    user = models.ForeignKey(User)
```

Comment:
This should be `user = models.ForeignKey(User, on_delete=models.CASCADE)`. This change should make the deprecation warnings in the current Django version go away.